src/Config.js

```diff
@@ -111,6 +113,7 @@ export class Config {
     this.validateIdempotencyOptions(idempotencyOptions);
     this.validatePagesOptions(pages);
     this.validateSecurityOptions(security);
+    this.validateSchemaOptions(security);
```

Comment:
I guess this should read `(schema)`? A test would have detected this. Can you add a few tests for valid / invalid configs?